### DISCUSSION TITLE
Is normal for a Sitecore project to not have a npm run dev equivalent?

### DISCUSSION THREAD

**Discussion by FerMPY** [Sitecore/jss#1552] [Help] [answered]
### Description

So I'm working on a reactjs project right now, and there is nothing like npm run dev, so what people do is just wait for the build to finish and then test the app as you probably can imagine this is a really bad dev experience. 
I have been going through Sitecore JSS docs but it is still not clear to me what I need to run it in dev mode.
I would love it if anyone can give a little bit more insight into this or at least point me in the right direction.

**Comment by illiakovalenko** [comment, marked as answer]
We have a "jss start:connected" command that you can use, it's a dev environment

**Reply by illiakovalenko** [reply]
See more info in our [doc](https://doc.sitecore.com/xp/en/developers/hd/21/sitecore-headless-development/start-a-jss-app-in-connected-mode.html)
